src/mpyl/cli/projects.py

```diff
@@ -99,24 +100,45 @@ def show_project(obj, name):
     )
 
 
-@projects.command(help="Validate the yaml of found projects against their schema")
+@projects.command(help="Validate the yaml of changed projects against their schema")
+@click.option(
+    "--all",
+    "all_",
```

Comment:
Why do we need the underscore?

Reply:
because `all` is a reserved name in python apparently